_harp/eclipse_lyo/setup-an-oslc-provider-consumer-application.md

```diff
@@ -87,112 +162,107 @@ We require Java EE 6 or higher and JSTL:
 </dependency>
 ```
 
-### Lyo package dependencies
+## Lyo dependencies
 
-Lyo dependencies need:
+The minimal Lyo dependencies are:
 
 ```xml
 <dependency>
   <groupId>org.eclipse.lyo.oslc4j.core</groupId>
   <artifactId>oslc4j-core</artifactId>
-  <version>${version.lyo.core}</version>
+  <version>${version.lyo}</version>
 </dependency>
 <dependency>
   <groupId>org.eclipse.lyo.oslc4j.core</groupId>
   <artifactId>oslc4j-jena-provider</artifactId>
-  <version>${version.lyo.core}</version>
-</dependency>
-<dependency>
-  <groupId>org.eclipse.lyo.oslc4j.core</groupId>
-  <artifactId>oslc4j-wink</artifactId>
-  <version>${version.lyo.core}</version>
+  <version>${version.lyo}</version>
 </dependency>
 <dependency>
   <groupId>org.eclipse.lyo.oslc4j.core</groupId>
   <artifactId>oslc4j-json4j-provider</artifactId>
-  <version>${version.lyo.core}</version>
+  <version>${version.lyo}</version>
 </dependency>
 ```
 
-### OAuth support
+## OSLC OAuth support
 
-Finally, your adaptor might need to includes the experimental support for OAuth:
+If your adaptor needs to support OAuth, include the following:
 
 ```xml
 <dependency>
   <groupId>org.eclipse.lyo.server</groupId>
   <artifactId>oauth-core</artifactId>
-  <version>${version.lyo.server}</version>
+  <version>${version.lyo}</version>
 </dependency>
 <dependency>
   <groupId>org.eclipse.lyo.server</groupId>
   <artifactId>oauth-consumer-store</artifactId>
-  <version>${version.lyo.server}</version>
-  <exclusions>
-    <exclusion>
-      <groupId>org.slf4j</groupId>
-      <artifactId>slf4j-log4j12</artifactId>
-    </exclusion>
-  </exclusions>
+  <version>${version.lyo}</version>
 </dependency>
 <dependency>
   <groupId>org.eclipse.lyo.server</groupId>
   <artifactId>oauth-webapp</artifactId>
-  <version>${version.lyo.server}</version>
+  <version>${version.lyo}</version>
   <type>war</type>
 </dependency>
 ```
 
-> Exclusions are due to the [Bug 513477](https://bugs.eclipse.org/bugs/show_bug.cgi?id=513477)
-
-> OAuth is enabled by default. If you want to disable it, open the generated `Application` class and do the following changes:
->
-> * comment out line `RESOURCE_CLASSES.add(Class.forName("org.eclipse.lyo.server.oauth.webapp.services.ConsumersService"));`
-> * comment out line `RESOURCE_CLASSES.add(Class.forName("org.eclipse.lyo.server.oauth.webapp.services.OAuthService"));`
-> * change `catch (ClassNotFoundException e)` to `catch (Exception e)`
+To support OAuth, you will need to add the following JAX-RS Providers to your Application (the `javax.ws.rs.core.Application` subclass)
+```java
+RESOURCE_CLASSES.add(Class.forName("org.eclipse.lyo.server.oauth.webapp.services.ConsumersService"));
+RESOURCE_CLASSES.add(Class.forName("org.eclipse.lyo.server.oauth.webapp.services.OAuthService"));
+```
+## OSLC Client support
 
-### OSLC Client support
+If your adaptor also needs to consume resources from another adaptor, a dependency to the OSLC client package is needed:
```

Comment:
I am trying to use OSLC Server instead unless I talk about LyoD

Reply:
true. I made the change throughout teh document and not just here.
I kept using "adaptor" when it was part of variables. Otherwise, the change might require more debugging.